hack/test-update-storage-objects.sh

```diff
@@ -103,7 +103,7 @@ echo "${ETCD_VERSION}" > "${ETCD_DIR}/version.txt"
 
 # source_file,resource,namespace,name,old_version,new_version
 tests=(
-test/e2e/testing-manifests/rbd-storage-class.yaml,storageclasses,,slow,v1beta1,v1
```

Comment:
was the `,,` a bug?

Reply:
I agree, I'm concerned this may not produce equivalent behavior, can you explain what's going on here?

Reply:
this concern still hasn't been addressed